src/components/Layouts/Header.jsx

```
@@ -31,15 +32,20 @@ export const Header = () => {
                   <span className="self-center text-2xl font-semibold whitespace-nowrap dark:text-white">E-STORE</span>
               </Link>
               <div className="flex items-center relative">
-                  <span onClick={() => setDarkMode(!darkMode)} className="cursor-pointer text-xl text-gray-700 dark:text-white mr-5 bi bi-gear-wide-connected"></span>
+                  // TODO: Re-enable dark mode toggle
```

Biome (2.3.13)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)